app/assets/stylesheets/components/_navbar.scss

```diff
@@ -2,7 +2,7 @@
   padding: 0;
 }
 
-.navbar-rv a {
+.navbar-rv a, .mobile-menu-rv a {
```

Comment:
when applying the same style to several classes or elements, I'd rather display it like that :
.navbar-rv a, 
.mobile-menu-rv a {
   style
}
for lisibility purpose